src/ChatFooter.js

```diff
@@ -16,16 +16,18 @@ export default class ChatFooter extends React.Component {
       const { scope } = this.props;
 
       if (scope === "round") {
-        const { stage, player } = this.props;
+        const { stage, player, chatId } = this.props;
         stage.append("chat", {
           text,
           playerId: player._id,
+          chatId,
         });
       } else {
-        const { game, player } = this.props;
+        const { game, player, chatId } = this.props;
```

Comment:
I am not sure I like `chatId` as a scoping name, as it sounds like there's a special chat object in the DB, which there isn't, which could cause confusion.

Reply:
Good point, it's not structurally parallel to the `playerId`, which has its own object.